### PR TITLE
Add requested links to the sidebar footer.

### PR DESCRIPTION
**Pull request by danoc**

### CODE REVIEW

File: www/src/components/container/side-nav.jsx [hunk 1]
```diff
@@ -9,7 +9,26 @@ import { ScrollMarkerLink } from 'react-scroll-marker';
 import styles from './side-nav.module.scss';
 
 const SideNav = ({ children }) => (
-    <ul className={`flex-1 overflow-y-auto pb3 ${styles.sideNav}`}>{children}</ul>
+    <div className="flex-1 flex overflow-y-auto flex-column">
+        <ul className={`pb3 ${styles.sideNav}`}>{children}</ul>
+        <ul className="mt-auto ph3 pv3 flex">
+            <li className={styles.sideNavBottomFooterLinks}>
+                <Text
+                    href="https://github.com/thumbtack/thumbprint/blob/master/LICENSE"
```
Comment (kevinmbeaulieu) [inline www/src/components/container/side-nav.jsx:17]: This link isn't working for me in the deploy preview- is that just a quirk of the preview, or a real issue?

File: www/src/components/container/side-nav.jsx [hunk 2]
```diff
@@ -9,7 +9,26 @@ import { ScrollMarkerLink } from 'react-scroll-marker';
...
+                    className="black"
+                    size={2}
+                    elementName="a"
+                >
+                    License
+                </Text>
+            </li>
+            <li className={styles.sideNavBottomFooterLinks}>
+                <Text href="https://wwww.thumbtack.com/" className="black" size={2} elementName="a">
+                    thumbtack.com
```
Comment (kevinmbeaulieu) [inline www/src/components/container/side-nav.jsx:27]: Also maybe rename this to "Thumbtack, Inc." to match the footer on www.thumbtack.com?

File: www/src/components/container/side-nav.jsx (hunk 1 above)
Comment (danoc) [reply www/src/components/container/side-nav.jsx:17]: Whoops, I did it wrong. Fixed.

File: www/src/components/container/side-nav.jsx (hunk 2 above)
Comment (danoc) [reply www/src/components/container/side-nav.jsx]: Good idea, done. Also, I changed `wwww` to `www`. :derp:

**Review by kevinmbeaulieu — APPROVED** [review]
Looks great!

**Pull request merged**